src/robot.cpp

```diff
@@ -20,7 +20,7 @@ void Robot::speakCallback(const std_msgs::String::ConstPtr& msg)
 
 // ----------------------------------------------------------------------------------------------------
 
-Robot::Robot(std::string name, Id id)
+Robot::Robot(std::string name, Id id, bool disable_speedcap, bool uncertain_odom) : base(disable_speedcap, uncertain_odom)
```

Comment:
```suggestion
Robot::Robot(std::string name, Id id, bool disable_speedcap = false, bool uncertain_odom= false) : base(disable_speedcap, uncertain_odom)
```

Reply:
No, default values should be ONLY defined in the header.